src/components/ColorText/ColorText.scss

```diff
@@ -0,0 +1,42 @@
+@import '../variables';
+@import '../../../styles/mixins';
+
+$block: '.#{$ns}color-text';
+
+$textColorMap: (
+    'inherit' inherit,
```

Comment:
I guess you don't need component for this

Reply:
Why not? In my practice it sometimes helps.